Pickers/AbstractActionSheetPicker.m

```diff
@@ -709,7 +709,18 @@ - (void)presentActionSheet:(SWActionSheet *)actionSheet {
 
 - (void)configureAndPresentPopoverForView:(UIView *)aView {
     UIViewController *viewController = [[UIViewController alloc] initWithNibName:nil bundle:nil];
-    viewController.view = aView;
+    
+    if (@available(iOS 11, *)) {
```

Comment:
Please, wrap it in ` #if __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000` , since we want to support older xcode versions for a while. for discussion, please look at end of PR
https://github.com/skywinder/ActionSheetPicker-3.0/pull/421